CriteoPublisherSdk/Sources/Criteo.m

```diff
@@ -35,6 +35,8 @@
 
 @implementation Criteo
 
+@dynamic childDirectedTreatment;
```

Comment:
why do you need a dynamic ?

Reply:
The `@dynamic` tells the compiler to not create an instance variable. This is appropriate here because the actual value is stored in the `BidManager` class. Having both a `getter` and a `setter` is the same thing as `@dynamic`. However, if someone were to remove either the `getter` or the `setter` and the `@dynamic` was not present it could result in the `childDirectedTreatment` property to not be passed to the `BidManager`.